src/antsibull/ansible_core.py

```diff
@@ -66,7 +66,7 @@ async def _get_json(self, query_url: str) -> t.Dict[str, t.Any]:
     @lru_cache(None)
     async def get_release_info(self) -> t.Dict[str, t.Any]:
         """
-        Retrieve information about releases of the ansible-base/ansible-core package from pypi.
+        Retrieve information about releases of the ansible-core/ansible-core package from pypi.
```

Comment:
If we're only going to mention ansible-core, then we can simplify this. Alternatively, we could reverse it, to read `ansible-core/ansible-base`.
```suggestion
        Retrieve information about releases of the ansible-core package from pypi.
```

Reply:
This function returns both ansible-base and ansible-core releases. I first thought about still just mentioning ansible-core, but in a function which explicitly supports both I thought it's better to also mention both of them.